ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -199,7 +199,7 @@ function copy_kubeconfig_minimal() {
   echo 'api available'
 
   echo 'waiting for bootstrap to complete'
-  openshift-install --dir="${dir}" wait-for bootstrap-complete &
+  ${INSTALL} --dir="${dir}" wait-for bootstrap-complete &
```

Comment:
can we call it as a more meaningful name, maybe something like: ${INSTALLER_BINARY}